### DISCUSSION TITLE
Configuring babel in VSCode to allow arrow function for class setter/getter in TypeScript

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [babel/babel#12242] [Q&A]
I've spent the whole day searching the internet for a concrete answer. The only close I got it was about babel stage 2 that allows this, but I don't find a good tutorial how to set this up.

 I need this:
```ts
class MyClass {
get prop() => this._prop;
}
```

Or this (even better):

```ts
class MyClass {
get prop => this._prop;
}
//no parameters list because it's a getter
```
Dart already allows this.

I know I'm close, but still a little lost here. Thank you!? 🙏

**Comment by nicolo-ribaudo** [comment]
Babel doesn't support this.

Babel only supports ECMAScript proposals (https://github.com/tc39/proposals), TypeScript, Flow and JSX. Any other syntax is out-of-scope for the project.
